addon/helpers/intersect.js

```diff
@@ -9,9 +9,9 @@ export function intersect([...arrays]) {
   let results = confirmedArrays.pop().filter(candidate => {
     for (let i = 0; i < arrays.length; i++) {
       let found = false;
-      let array = arrays[i];
+      let array = emberArray(arrays[i]);
       for (let j = 0; j < array.length; j++) {
-        if (array[j] === candidate) {
+        if (array.objectAt(j) === candidate) {
```

Comment:
One question @gmurphey before merging - what makes this code as is on master not able to work with Ember arrays?

Reply:
ping!